tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_Contacts_RTL.py

```diff
@@ -0,0 +1,64 @@
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.contacts.app import Contacts
+
+class TestContactsRTL(GaiaImageCompareTestCase):
+
+    def setUp(self):
+       GaiaImageCompareTestCase.setUp(self)
+
+       self.push_resource('IMG_0001.jpg')
+
+    def test_contacts_app(self):
+
+        contacts = Contacts(self.marionette)
+        contacts.launch()
+        self.take_screenshot('contacts')
+        view_settings = contacts.tap_settings()
+        self.take_screenshot('contacts-setting')
+        view_settings.tap_import_contacts()
+        self.take_screenshot('contacts-import')
+        view_settings.tap_import_from_sim()
+        self.take_screenshot('contacts-sim_import')
```

Comment:
Ok, will try with email I think I need the test account if you have it @mwargers or @npark-mozilla